msal/src/main/AndroidManifest.xml

```diff
@@ -21,5 +21,9 @@
         <activity
             android:name="com.microsoft.identity.client.BrowserTabActivity"
             android:exported="true" />
+
+        <activity
+            android:name="com.microsoft.identity.client.internal.controllers.BrokerActivity"
+            android:exported="true" />
```

Comment:
Can we drop a comment in here clarifying why/for-whom this `Activity` is exported?

Reply:
Additionally, we need to update our guidance such that developers declare this Activity in their application, or will manifest merging alone take care of this?

Reply:
Actually we use this internally, exported need not be true, was an oversight due to a copy/paste error. Fixed it